### DISCUSSION TITLE
[Feature request]  Is there a way for the patch to work with Citra on Switch?

### DISCUSSION THREAD

**Discussion by gagrecco** [leoetlino/project-restoration#179] [Q&A] [answered]
Im trying to run MM3D with the restoration patch and hud on Citra Switch. but Citra does not support mods. Is there a way to inject the files to the rom itself? maybe by decompiling it and building it again? If so how?

**Comment by leoetlino** [comment, marked as answer]
It should be possible (with https://github.com/SciresM/RomFS-Builder for example) but I've never done it so unfortunately I wouldn't be able to write a guide for you.

Keep in mind that if the version of Citra you're using is too old, you will also have to patch the executable manually.
